.editorconfig

```diff
@@ -0,0 +1,12 @@
+# based on the airbnb styleguide https://github.com/airbnb/javascript/blob/master/.editorconfig
```

Comment:
Works pretty well for me with vscode. Let's get agreement from one other person on this though.